.github/workflows/iroha2-dev-pr.yml

```diff
@@ -91,18 +102,23 @@ jobs:
             clang \
             llvm-dev
       - name: Run iroha tests with network mock
-        run: cargo test --features mock -- --ignored --test-threads=1
+        run: cargo test --features mock -- --ignored
         working-directory: iroha/test_network
 
-  test-deadlock_detection:
+  test-deadlock-detection:
```

Comment:
Did the highlighting break in GitHub? :smile:

Reply:
Nope, just consistancy)

Reply:
Ah I see `-` instead of `_`